docker/generic/start_proxy.py

```diff
@@ -177,6 +177,11 @@ def make_argparser():
         help=''' Maximum TLS protocol version for client side connection.
         Please refer to https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/auth/cert.proto#common-tls-configuration.
         ''')
+    
+    parser.add_argument('--enable_strict_transport_security', action='store_true',
```

Comment:
Can we validate that SSL is enabled (`--ssl_server_cert_path` is used) for this setting to apply? Otherwise, this flag would cause conflicting behavior. Not sure if that maintains backwards compatibility.

Reply:
Will follow up to see whether it is needed or not.